clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/securitygroup/SecurityGroupIngressConverter.groovy

```diff
@@ -75,7 +85,10 @@ class SecurityGroupIngressConverter {
         ipPermissions.add(newIpPermission)
       }
     }
-    new ConvertedIngress(ipPermissions, missing)
+    new ConvertedIngress(ipPermissions, new MissingSecurityGroups(
+      all: missing,
+      selfReferencing: missing.findAll { it.name == description.name }
```

Comment:
Should maybe probably add more equality checks here?

Reply:
account/userId maybe?